nodebuilder/tests/reconstruct_test.go

```diff
@@ -103,7 +103,17 @@ func TestFullReconstructFromLights(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), swamp.DefaultTestTimeout)
 
 	t.Cleanup(cancel)
-	sw := swamp.NewSwamp(t, swamp.WithBlockTime(btime))
+	sw := swamp.NewSwamp(t, swamp.WithBlockTime(btime), swamp.WithLogLevel(
+		[]string{
+			"bitswap_network",
```

Comment:
can you add "dagstore" and "dagstore/upgrader"? They also spam a lot of logs

Reply:
added as an example. Now *you* are the artist of your own logs 😄 

But, sure, will add dagstore